doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+## Getting started
+
+Your [Dogecoin Core installation](/INSTALL.md) is done, you now have `dogecoind` or `dogecoin-qt` executables available to run a node, and `dogecoin-cli`/`dogecoin-tx` tools but don't know how it works ?
+
+This tutorial will help you to go through the basics to use them !
+
+> **Warning :** For simplicity, explained commands assume executables are under `PATH` environment variable.  
+> If needed, specify their location by doing `PATH=$PATH:/path/to/executables` or use their path as command like :
+> ```bash
+> $ /path/to/dogecoin-cli [arguments ...]
+> ```
+
+### Table of contents
+
+1. [Run a dogecoin node](#run-a-dogecoin-node)
+2. [Introduction to JSON-RPC API](#introduction-to-json-rpc-api)
+    * [Create a wallet](#create-a-wallet)
+    * [Send a transaction](#send-a-transaction)
+    * [Inspect blocks and transactions](#inspect-blocks-and-transactions)
+3. [Node configuration](#node-configuration)
+    * [Mainnet, testnet and regnet](#mainnet-testnet-and-regnet)
+    * [RPC credentials](#rpc-credentials)
+    * [Ports](#ports)
+    * [Memory](#memory)
+    * [Mode debug](#mode-debug)
+
+## Run a dogecoin node
+
+To launch your node, you can choose `dogecoind` to run an headless server or `dogecoin-qt` for a server with a GUI.
+
+In a terminal, run :
+```bash
+# Headless node
+$ dogecoind -printtoconsole
```

Comment:
why `-printtoconsole` and not `-daemon`?

Reply:
Personal experience where I was confused about not having any output on the first time I runned a node. I find it more intuitive to display first some output and then introduce debug.log.

Why `-daemon` and not `&` ?

Reply:
because `-daemon` detaches from the user session and has the root process as its parent, whereas `&` has the terminal process as its parent. Meaning that if the terminal gets killed, dogecoind gets killed. What if the terminal gets killed because it ran OOM due to all the output you fed it with `-printtoconsole`?

Reply:
Don't you think it may be good to keep `-printtoconsole` for intro here, and use daemon in the example for configuration later ? Like it they will have both, with a first one showing some output, and finally a way to daemonize their node ?

Reply:
Changed to use `-daemon`. Let me know if it's ok with the current modification.

Reply:
Yes. Looks good!